src/TransitionGroup.js

```diff
@@ -16,8 +16,8 @@ const propTypes = {
    * `<TransitionGroup>` renders a `<div>` by default. You can change this
    * behavior by providing a `component` prop.
    * If you use React v16+ and would like to avoid a wrapping `<div>` element
-   * you can pass in `component={null}`. This is useful if the wrapping div
-   * borks your css styles.
+   * you can invoke a function and return `null` to `component`. This is useful 
```

Comment:
what function do you mean here?

Reply:
This issue came up here: https://github.com/reactjs/react-transition-group/issues/20

Passing `component={null}` is not what's intended here. What is intended is:

```
<TransitionGroup component={returnNull} />

function returnNull() {
return null;
}
```

Reply:
o I see, the note here is actually talking about a different thing. The comment in #20 refers to the v1 API and is a workaround for rendering only one item or nothing. in the current version (v2) there isn't any need for that workaround anymore because the `Transition` now exists for when you want to render one thing in and out.

 the docs here are correct for what they are saying, which is that TransitionGroup renders a `div` around your children, if you set it to `null` instead it will return just the children with no wrapper, something that is possible in react v16+

Reply:
Strange. This isn't working for me. In React v16+, it throws an error saying it expected a string, but received null.